compiler/src/main/java/com/squareup/anvil/compiler/InterfaceMerger.kt

```diff
@@ -159,7 +159,7 @@ internal class InterfaceMerger(
     if (excludedClasses.isNotEmpty()) {
       val intersect = supertypes.getAllSuperTypes()
         .toList()
-        .intersect(excludedClasses)
+        .intersect(excludedClasses.toSet())
```

Comment:
IDE showed this here (and at other places) and this makes sense, because then the potentially same elements don't need to be checked multiple times.

Reply:
Woah neat